packages/server-wallet/package.json

```diff
@@ -24,6 +24,7 @@
     "pg-connection-string": "^2.4.0",
     "pino": "6.2.0",
     "rxjs": "6.6.3",
+    "set-interval-async": "^2.0.3",
```

Comment:
This just makes test cleanup much easier. We can `await` `clearIntervalAsync` and be sure that the sync method is done before destroying.